web/test/testgrid-index.test.ts

```diff
@@ -22,10 +22,6 @@ describe('Testgrid Index page', () => {
     element = await fixture(html`<${tag}></${tag}>`);
   });
 
-  it('passes the a11y audit', async () => {
```

Comment:
QQ, any reason for removing this?

Reply:
Ah, gotcha! Commenting + TODO'ing sounds good to me if that's alright, I think we want to pass the accessibility checks but don't have to do it here.

Reply:
Adding TODOs now